src/Logging/src/SerilogDynamicLoggerCore/SerilogBuilderExtensions.cs

```diff
@@ -32,6 +33,12 @@ public static ILoggingBuilder AddSerilogDynamicConsole(this ILoggingBuilder buil
                 throw new InvalidOperationException("An IDynamicLoggerProvider has already been configured! Call 'AddSerilogDynamicConsole' earlier in program.cs (Before AddCloudFoundryActuators()) or remove duplicate IDynamicLoggerProvider entries.");
             }
 
+            var defaultConsoleDescriptor = builder.Services.FirstOrDefault(d => d.ImplementationType == typeof(ConsoleLoggerProvider));
+            if (defaultConsoleDescriptor != null)
+            {
+                builder.Services.Remove(defaultConsoleDescriptor);
+            }
+
```

Comment:
As-is, this seems like a change that might be too extreme for a patch release (actively removing something that wasn't added by our code). Can we make it opt-in and switch it to be default behavior down the road?

Reply:
We are just making it consistent with DynamicConsole -  https://github.com/SteeltoeOSS/Steeltoe/blob/4a396adf1069ccd1ad3bc8603719569579efae7b/src/Logging/src/DynamicLogger/DynamicLoggingBuilder.cs#L33
But if you feel strongly I can add a flag and allow opt in.

Reply:
I'm in favor of it being this way -- I'm just hesitating on adding it _in a patch release_ where it is on-by-default and there's no opt-out.

Reply:
Added an opt-out to preserve default logger.